android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/PwoDiscoveryService.java

```diff
@@ -63,6 +69,11 @@
 
   private static final String TAG = "PwoDiscoveryService";
   private static final String NOTIFICATION_GROUP_KEY = "URI_BEACON_NOTIFICATIONS";
+  private static final String SHARED_PREFS_FILE = "pwo_discovery_service_cache";
```

Comment:
Dunno...lemme check best practices...

I think it makes sense to keep it distinct, though we should be using better scoped names.

Let's do #465 first, and then I'll change this.